includes/Story_Renderer/HTML.php

```diff
@@ -229,6 +230,26 @@ protected function transform_html_start_tag() {
 		}
 	}
 
+	/**
+	 * Transform all a tags to add target and rel attributes.
+	 *
+	 * @since 1.1.0
+	 *
+	 * @return void
+	 */
+	protected function transform_a_tags() {
+		/* @var DOMElement $html The <a> element */
+		$hyperlinks = $this->document->getElementsByTagName( 'a' );
+		foreach ( $hyperlinks as $hyperlink ) {
+			if ( ! $hyperlink->getAttribute( 'target' ) ) {
+				$hyperlink->setAttribute( 'target', esc_attr( '_blank' ) );
```

Comment:
`esc_attr` is unnessary because `setAttribute` does the escaping already. Plus it's no user input we need to sanitize, just a static string.